### PR TITLE
Update banner text per USWDS 2.8.0 + translations

### PR DESCRIPTION
**Pull request by louh**
https://github.com/usdoj-crt/crt-portal-management/issues/1114, https://github.com/usdoj-crt/crt-portal-management/issues/1050

## What does this change?

[USWDS 2.8.0 updated the banner text](https://github.com/uswds/uswds/releases/tag/v2.8.0). See USWDS release notes for more infomration.

We previously updated the codebase for USWDS 2.8.0 in https://github.com/usdoj-crt/crt-portal/pull/984. We are now able to update the text content after receiving translations in all languages we support. _Please note that I am using the official Spanish translations provided by USWDS, not the Spanish translations that we've received from the FBI._

(Update: re Spanish translations. I checked the FBI translations and the USWDS translations. They're very similar. We'll stick with the USWDS translations for consistency with other US government websites in Spanish, but I feel comfortable saying that FBI was not off the mark on theirs.)

### English

Screenshot 2022-02-04 at 10-22-24 Contact the Civil Rights Division Department of Justice

### Spanish
Screenshot 2022-02-04 at 09-45-35 Comuníquese con la División de Derechos Civiles Departamento de Justicia

### Korean
Screenshot 2022-02-04 at 09-50-09 법무부 민권국 연락처

### Tagalog
Screenshot 2022-02-04 at 09-53-12 Makipag-ugnay sa Dibisyon sa mga Karapatang Sibil Kagawaran ng Katarungan

### Vietnamese
Screenshot 2022-02-04 at 09-55-55 Liên hệ với Ban Dân Quyền Bộ Tư pháp

### Simplified Chinese
Screenshot 2022-02-04 at 10-07-39 联系司法部民权司

### Traditional Chinese
Screenshot 2022-02-04 at 10-07-53 與民權科 司法部聯絡

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [x] Re-check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -2812,6 +2814,20 @@ msgstr ""
...
+#~ msgid ""
+#~ "Federal government websites often end in .gov or .mil. Before sharing "
+#~ "sensitive information, make sure you’re on a federal government site."
+#~ msgstr ""
+#~ "Las páginas web federales suelen acabar en .gov o .mil. Antes de "
+#~ "compartir información confidencial, compruebe que se encuentra en un "
+#~ "sitio del gobierno federal."
+
+#~ msgid "The site is secure."
+#~ msgstr "Este sitio es seguro."
```
Comment (billyhunt) [inline crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po:2829]: Any idea why the compile just commented this out?  Should we keep it?

File: crt_portal/cts_forms/templates/partials/banner/usa_banner_lock_icon.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<span class="icon-lock"><svg xmlns="http://www.w3.org/2000/svg" width="52" height="64" viewBox="0 0 52 64"
+    class="usa-banner__lock-image" role="img"
+    aria-labelledby="banner-lock-title-default banner-lock-description-default" focusable="false">
+    <title id="banner-lock-title-default">Lock</title>
+    <desc id="banner-lock-description-default">A locked padlock</desc>
+    <path fill="#000000" fill-rule="evenodd"
+      d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
+  </svg></span>
```
Comment (billyhunt) [inline crt_portal/cts_forms/templates/partials/banner/usa_banner_lock_icon.html:8]: Is it better to inline the SVG or should we put it in an image directory?

File: crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po (hunk 1 above)
Comment (louh) [reply crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po:2829]: It happened automatically during `makemessages`, and my guess is that it does this to allow fuzzy translations based on previously translated text even if the original stops existing in exact form. I don't think we have anything to lose by removing it.

**Review by billyhunt — APPROVED** [review]
Looks good to me!

**Pull request merged**